### PR TITLE
make final annotations field in TestPlanElement

### PR DESCRIPTION
**Pull request by Grigory-Rylov**
fixed NPE in TestPlanElement.annotationsAsString() when test without annotation.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,5 +1,5 @@
 buildscript {
-    def kotlin_version = '1.3.21'
```
Comment (Nexen23) [inline build.gradle]: Why downgrade?

File: build.gradle [hunk 2]
```diff
@@ -39,20 +39,20 @@ repositories {
 }
 
 dependencies {
-    def kotlin_version = '1.3.21'
-    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
-    implementation 'org.apache.logging.log4j:log4j-api:2.6.1'
-    implementation 'org.apache.logging.log4j:log4j-core:2.6.1'
-    implementation 'com.google.code.gson:gson:2.8.5'
+    def kotlin_version = '1.2.61'
+    compile "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlin_version"
```
Comment (Nexen23) [inline build.gradle]: why use deprecated gradle options?

File: src/test/java/com/github/grishberg/tests/planner/InstrumentTestLogParserTest.java [hunk 3]
```diff
@@ -20,19 +20,11 @@
 @RunWith(MockitoJUnitRunner.class)
 public class InstrumentTestLogParserTest {
     private static final String TEST_LINE = "test line";
+    private static final String TEST_ELEMENT_WITHOUT_ANNOTATION = "TestPlanElement{methodName=" +
+            "'ignoredTestTabletButton2', className='com.github.grishberg.instrumentaltestsample." +
+            "TabletTest', type=METHOD, annotations=[]}";
```
Comment (Nexen23) [inline src/test/java/com/github/grishberg/tests/planner/InstrumentTestLogParserTest.java]: Seems test name now isn't comply with its purposes

**Comment by Nexen23** [comment]
Also builds failed

File: build.gradle (hunk 2 above)
Comment (Grigory-Rylov) [reply build.gradle]: changes in gradle-5.1.1 brokes classes for supporting https://github.com/Grigory-Rylov/Test-Report-with-screenshots

File: src/test/java/com/github/grishberg/tests/planner/InstrumentTestLogParserTest.java (hunk 3 above)
Comment (Grigory-Rylov) [reply src/test/java/com/github/grishberg/tests/planner/InstrumentTestLogParserTest.java]: how should I name it?

File: build.gradle (hunk 1 above)
Comment (Grigory-Rylov) [reply build.gradle]: for compatibility with gradle-wrapper-3.1

File: src/test/java/com/github/grishberg/tests/planner/InstrumentTestLogParserTest.java (hunk 3 above)
Comment (Nexen23) [reply src/test/java/com/github/grishberg/tests/planner/InstrumentTestLogParserTest.java]: SimpleTest/NonAnnotatedTest/e.t.c

**Pull request merged**